src/sql/platform/accounts/common/accountActions.ts

```diff
@@ -150,3 +151,31 @@ export class RefreshAccountAction extends Action {
 		}
 	}
 }
+
+/**
+ * Action to sign out of GitHub Copilot account
+ */
+export class GitHubCopilotSignOutAction extends Action {
+	public static ID = 'account.github.copilot.sign.out';
+	public static LABEL = localize('githubCopilotSignOut', "Sign out of GitHub Copilot");
+	public account?: azdata.Account;
+
+	constructor(
+		private _account: azdata.Account,
+		@IAccountManagementService private _accountManagementService: IAccountManagementService,
```

Comment:
nit: If we're going to use readonly we should be consistent within a file

```suggestion
		@IAccountManagementService private readonly _accountManagementService: IAccountManagementService,
```